torchvision/datapoints/_datapoint.py

```diff
@@ -14,6 +14,13 @@
 
 
 class Datapoint(torch.Tensor):
+    """[Beta] Base class for all datapoints.
+
+    You probably don't want to use this class unless you're defining your own
+    custom Datapoints. See
+    :ref:`sphx_glr_auto_examples_plot_custom_datapoints.py` for details.
```

Comment:
That sounds a little negative. Can't we just link the gallery for an example?

Reply:
Happy to consider alternative suggestions but I think we should drive most readers away from this class. My main goal here is that I don't want to scare them and make them believe they need to understand what this is/does just to use v2.